app/controllers/IdentityController.scala

```diff
@@ -1,9 +1,10 @@
 package controllers
 
 import actions.CustomActionBuilders
-import com.typesafe.scalalogging.LazyLogging
 import io.circe.Decoder
 import io.circe.generic.semiauto.deriveDecoder
+import monitoring.SafeLogger
+import monitoring.SafeLogger._
```

Comment:
as you import `SafeLogger._` you can actually change `SafeLogger.info(` to just `info(` if you prefer that way